src/analysis/complex/removable_singularity.lean

```diff
@@ -126,4 +126,33 @@ lemma tendsto_lim_of_differentiable_on_punctured_nhds_of_bounded_under {f : ℂ
   tendsto f (𝓝[≠] c) (𝓝 $ lim (𝓝[≠] c) f) :=
 tendsto_lim_of_differentiable_on_punctured_nhds_of_is_o hd hb.is_o_sub_self_inv
 
+lemma two_pi_I_inv_smul_circle_integral_sub_inv_smul_of_differentiable
```

Comment:
```suggestion
lemma two_pi_I_inv_smul_circle_integral_sub_sq_inv_smul_of_differentiable
```

Reply:
I would also suggest adding a docstring here as it doesn't really match the "tone" which is always working in `𝓝[≠] c`